samples/mcp9808.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/kidoman/embd"
+	_ "github.com/kidoman/embd/host/rpi"
+	"github.com/kidoman/embd/sensor/mcp9808"
+	"github.com/stianeikeland/go-rpio"
```

Comment:
I apologize, I was getting a weird "Not Implemented" error with embd when I was trying to do a read on the pin and forgot to go back and trace what I was doing wrong.  I'll get rid of that dependency.